### PR TITLE
[SERV-1098] Add docker compose file and update name of service

### PR DESCRIPTION
**Pull request by angelahuqing**

### CODE REVIEW

File: testcontainer_test.go [hunk 1]
```diff
@@ -64,7 +64,7 @@ func TestApp(t *testing.T) {
 	// Any requeset body will work since POST requests are not currently allowed
 	requestBody := []byte(`{"key": "value"}`)
 
-	resp, err = client.Post("http://" + host + ":" + strconv.Itoa(port.Int()) + "/", "application/json", bytes.NewBuffer(requestBody))
+	resp, err = client.Post("http://"+host+":"+strconv.Itoa(port.Int())+"/", "application/json", bytes.NewBuffer(requestBody))
```
Comment (ksclarke) [inline testcontainer_test.go:67]: I'm curious why you removed the spaces. It seems more readable with them(?)

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+services:
+  metadata-extraction-service:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8888:8888"
```
Comment (ksclarke) [inline docker-compose.yml:7]: Add a trailing empty line so GitHub doesn't warn us about that.

File: testcontainer_test.go (hunk 1 above)
Comment (angelahuqing) [reply testcontainer_test.go:67]: Yes, I agree. VSCode has an autosave format that does it automatically.

**Review by DRickard — APPROVED** [review]

File: testcontainer_test.go (hunk 1 above)
Comment (ksclarke) [reply testcontainer_test.go]: Is that configurable?

**Review by ksclarke — APPROVED** [review]

**Pull request merged**